src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,9 @@
+package duke.parser;
+
+public class Parser {
```

Comment:
Having the parser be the one putting in the work to decode the full command might be a better Separation of Concern in the OOP paradigm